package.json

```diff
@@ -30,7 +30,7 @@
     "@testing-library/react": "^12.1.2",
     "@types/jest": "^27.0.2",
     "@types/react": "^17.0.2",
-    "@types/react-dom": "^17.0.1",
+    "@types/react-dom": "^17.0.2",
```

Comment:
I was having issues with snapshot tests for the `Dropdown` component, an error about invalid hooks.  When I searched the error, possible cause were `react-dom` and `react` being different versions.  Aligning versions seemed to fix the issue, so I'm committing this.